recipes/mpiFFT4py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mpiFFT4py" %}
```

Comment:
I could be wrong, but this is the first case we have merged something with a package name that contains capital letters. There has been an assumption that lower case letters are used in packages/feedstocks that is baked into a lot of code. Before we start digging into it, I think a fair question is do we wish to allow packages to have capital letters in them or not?

cc @conda-forge/core @conda-forge/staged-recipes